woocommerce/woocommerce
# Warn that jquery-cookie and prettyPhoto are deprecated scripts

### All Submissions:






### Changes proposed in this Pull Request:

prettyPhoto and jquery-cookie are deprecated scripts. They were informally marked as deprecated with a code comment approx 5 years ago. This PR adds a hook to notify users enqueueing these scripts that they will be removed in a future version.

Since we never enqueue these scripts ourselves I think its reasonable not to treat our registering them as a public API but to be safe these warnings will give any user still using them a chance to fix that issue.








### How to test the changes in this Pull Request:



1. Checkout this PR in your Wordpress dev environment
2. Add some lines of code that execute any time after `WC_Frontend_Scripts` is executed. These lines of code should enqueue any one of the scripts or styles that this function warns about.
3. You should see in your debug log that warnings for the scripts you enqueued are displayed.



### Other information:




### FOR PR REVIEWER ONLY:
